spec/services/candidates/registrations/contact_information_spec.rb

```diff
@@ -18,10 +18,35 @@
     it { is_expected.to validate_presence_of :full_name }
     it { is_expected.to validate_presence_of :email }
     it { is_expected.to validate_presence_of :building }
-    it { is_expected.to validate_presence_of :street }
-    it { is_expected.to validate_presence_of :town_or_city }
-    it { is_expected.to validate_presence_of :county }
     it { is_expected.to validate_presence_of :postcode }
     it { is_expected.to validate_presence_of :phone }
+
+    context 'phone is present' do
+      VALID_NUMBERS = ['01434 634996', '+441434634996', '01234567890'].freeze
+      INVALID_NUMBERS = ['7', 'q', '+4414346349'].freeze
```

Comment:
Would like to see a blank number being tested as invalid here?

Reply:
That should get picked up by the presence validation on phone number but added spec for completeness anyway 👍